data/categories.txt

```diff
@@ -7186,7 +7190,7 @@ pl:^Wejście
 pt:^Entrada
 ro:^Intrare
 es:^Entrada
-sv:^Entré
+sv:^Entré|ingång
```

Comment:
The rest seem ok, but here I'd like to double check.
The symbol '^' in this file marks the translation that will be shown in the UI. Is it true that "Entré" is more commonly used than "Ingång"?